site/src/pages/components/link-area-delegation-explainer.mdx

```diff
@@ -0,0 +1,191 @@
+---
+menu: Active Proposals
+name: Link Area Delegation (Explainer)
+##path: /components/...
+##pathToResearch: /components/...
+layout: ../../layouts/ComponentLayout.astro
+---
+
+- Authors and contributors: [@noamr](https://github.com/noamr), [@scottaohara](https://github.com/scottaohara), [@nmn](https://github.com/nmn), [@jakearchibald](https://github.com/jakearchibald)
+- Last updated: October 16, 2024
+- Feedback: https://github.com/openui/open-ui/issues/
+
+{/* START doctoc generated TOC please keep comment here to allow auto update */}
+{/* DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE */}
+
+## State of this explainer
+We are currently incubating and exploring a new idea. Chime in!
+
+## Overview
+Historically and for very good reasons, interactive elements on the web cannot be nested within other interactive elements, including hyperlinks.
+Interactive elements not allowing descendent interactive elements is a well established rule in HTML for various interactive elements - where descendent interactive elements or elements with a `tabindex` attribute are specifically called out as invalid (see [the `a` element's content model](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element). Additionally, the [ARIA in HTML guideline]([https://w3c.github.io/html-aria/](https://w3c.github.io/html-aria/#allowed-descendants-of-aria-roles)) specification provides further guidance to discourage the use of interactive elements as descendent of ARIA roles which are themselves considered "widget" (interactive) roles.
```

Comment:
```suggestion
This restriction on nested interactive elements is a well established rule in HTML - where descendent interactive elements or elements with a `tabindex` attribute within other interactive elements are specifically called out as invalid (see [the `a` element's content model](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element). Additionally, the [ARIA in HTML guideline]([https://w3c.github.io/html-aria/](https://w3c.github.io/html-aria/#allowed-descendants-of-aria-roles)) further discourages the use of interactive elements as descendent of ARIA roles which are themselves considered "widget" (interactive) roles.
```

Reply:
thanks.  going to rework this a bit further in an updated commit i'm working on